services/github/auth/admin.spec.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+const { expect } = require('chai')
+const sinon = require('sinon')
+const Camp = require('camp')
+const fetch = require('node-fetch')
+const config = require('../../../lib/test-config')
+const serverSecrets = require('../../../lib/server-secrets')
+const { setRoutes } = require('./admin')
+
+function createAuthHeader({ username, password }) {
```

Comment:
It's a bit annoying this must be done manually. We may want to consider a different request library like got or axios.